tools/configure/cmdArgs.ml

```diff
@@ -0,0 +1,199 @@
+    "<dir> Set installation directory to <dir> (absolute path required)";
+  "-no-ask", arg_set (fun p -> { p with interactive = false }),
+    " Don't ask questions / print variables during configure [questions will be filled with defaults]";
+  "-libdir", arg_string_option (fun p libdir -> { p with libdir }),
+    "<dir> Where to install lib files";
+  "-configdir", arg_string_option (fun p configdir -> { p with configdir }),
+    "<dir> Where to install config files";
+  "-datadir", arg_string_option (fun p datadir -> { p with datadir }),
+    "<dir> Where to install data files";
+  "-mandir", arg_string_option (fun p mandir -> { p with mandir }),
+    "<dir> Where to install man files";
+  "-docdir", arg_string_option (fun p docdir -> { p with docdir }),
+    "<dir> Where to install doc files";
+  "-arch", arg_string_option (fun p arch -> { p with arch }),
+    "<arch> Specifies the architecture";
+  "-natdynlink", arg_bool (fun p natdynlink -> { p with natdynlink }),
+    "(yes|no) Use dynamic loading of native code or not";
+  "-coqide", arg_ide (fun p coqide -> { p with coqide }),
+    "(opt|byte|no) Specifies whether or not to compile CoqIDE";
+  "-nomacintegration", arg_set (fun p -> { p with macintegration = false}),
+    " Do not try to build CoqIDE MacOS integration";
+  "-browser", arg_string_option (fun p browser -> { p with browser }),
+    "<command> Use <command> to open URL %s";
+  "-with-doc", arg_bool (fun p withdoc -> { p with withdoc }),
+    "(yes|no) Compile the documentation or not";
+  "-byte-only", arg_set (fun p -> { p with byteonly = true }),
+    " Compiles only bytecode version of Coq";
+  "-annot", arg_set (fun p -> { p with annot = true }),
+    " Dumps ml text annotation files while compiling Coq (e.g. for Tuareg)";
+  "-bin-annot", arg_set (fun p -> { p with bin_annot = true }),
+    " Dumps ml binary annotation files while compiling Coq (e.g. for Merlin)";
+  "-bytecode-compiler", arg_bool (fun p bytecodecompiler -> { p with bytecodecompiler }),
+    "(yes|no) Enable Coq's bytecode reduction machine (VM)";
+  "-native-compiler", arg_native (fun p nativecompiler -> { p with nativecompiler }),
+    "(yes|no|ondemand) Compilation to native code for conversion and normalization
+     yes: -native-compiler option of coqc will default to 'yes', stdlib will be precompiled
+     no: no native compilation available at all
+     ondemand (default): -native-compiler option of coqc will default to 'ondemand', stdlib will not be precompiled";
+  "-coqwebsite", arg_string (fun p coqwebsite -> { p with coqwebsite }),
+    " URL of the coq website";
+  "-warn-error", arg_bool (fun p warn_error -> { p with warn_error }),
+    "(yes|no) Make OCaml warnings into errors (default no)";
+  "-profile", arg_profile, Profiles.doc
+]
+
+let parse_args () =
+  Arg.parse
+    args_options
+    (fun s -> raise (Arg.Bad ("Unknown option: "^s)))
+    "Available options for configure are:";
+  !prefs
+
+(* Support don't ask *)
```

Comment:
What was the meaning of this comment?

Reply:
It is coming from 615c29be047eafac1ec1a9a853afe5a577873771 , and it means "don't ask me what the code involved with -prefix is doing`